### ISSUE TITLE
[A11y] Desktop Explore Collection Grid: Find a Collection Search Box

### ISSUE THREAD

**Issue by azle** [OregonDigital/OD2#1794]
### Operating System

MacOS

### Browser

Chrome

### Affected page

Explore Collections

### Admin user?

- [ ] Yes

### Issue Description

Explore Collection - Grid: https://staging.oregondigital.org/explore_collections/all?locale=en&view=masonry

The "Find a Collection" search box has a duplicate-id-aria.

Failed Instance:
duplicate-id-aria: ensures every ID attribute value used in aria and in labels is unique

Path: form[action="\/explore_collections\/all"]

Snippet:
`<form class="form-horizontal search-form" id="explore-collection-search" role="search" action="/explore_collections/all" accept-charset="UTF-8" method="get">`

### Suggested Fix

Fix the following:
Document has multiple elements referenced with ARIA with the same id attribute: explore-collection-search

**Comment by CGillen** [comment]
Fixed by #1965
